pkg/cmd/openshift-controller-manager/controller/image.go

```diff
@@ -63,51 +54,41 @@ func (c *ImageTriggerControllerConfig) RunController(ctx ControllerContext) (boo
 			Reactor:   &triggerdeploymentconfigs.DeploymentConfigReactor{Client: appsClient.Apps()},
 		},
 	}
-	if !c.HasBuilderEnabled {
```

Comment:
@deads2k was this even logically correct before? `if has not builder enabled, then add trigger for build configs` sounds weird to me?

Reply:
i guess not :-) and nobody noticed this since 3.6... shame on me and shame on @smarterclayton who manually merged it ;-)

Reply:
nvmd. false alarm, this code is unfortunate but it works because we check if the feature is disabled when we initialize this struct (the negation should have been there to make this more clear...)